sequoia/settings/assumptions/continual.py

```diff
@@ -433,7 +434,14 @@ def step(self, action):
         else:
             done = bool(done)
 
-        done = self._after_step(observation_for_stats, reward_for_stats, done, info)
+        # If the environment is a multi-agent environment, the reward_for_stats will be a vector of agent rewards,
+        # which must be condensed to a single scalar reward
+        if isinstance(reward_for_stats, np.ndarray):
```

Comment:
This should check if the env is a multi agent RL environment, not if the env is vectorized.
The Monitor wrapper from Gym doesn't (or didn't) support vectorized environments. So technically this check works, since the only vectorized test env is yours. But still. Hope this makes some sense.

Reply:
I think this is pretty much resolved, but I could be wrong.